main.go

```diff
@@ -59,6 +64,20 @@ func main() {
 		prometheus.MustRegister(certExporter)
 	}
 
+	// Monitor expiry of secrets of type kubernetes.io/tls
+	if monitorSecrets {
+		c := secret.DefaultConfig()
+		if namespaceWhitelist != "" {
```

Comment:
the `flag` thingie we use supports string slices as well, so you don't have to deal with parsing,